src/llrt-function.ts

```diff
@@ -38,6 +38,13 @@ export interface LlrtFunctionProps extends NodejsFunctionProps {
    * @default LlrtBinaryType.STANDARD
    */
   readonly llrtBinaryType?: LlrtBinaryType;
+
+  /**
+   * A custom relative directory to find the LLRT "bootstrap" binary in.
+   *
+   * @default - The LLRT binary is downloaded from GitHub and cached in the .tmp directory.
+   */
+  readonly llrtCustomBinaryDirectory?: string;
```

Comment:
How about renaming it to `llrtBinaryPath` and take binary file path instead of directory path? Such name indicates more clearly that users have to explicitly prepare the binary.